### PR TITLE
UI tests - Add PS versions from 1.7.7.0 ~ 1.7.7.8 to 1.7.8.11 and 8.2.0 on PHP 7.1 ~ 7.3

### PR DESCRIPTION
**Pull request by nesrineabdmouleh**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Add PS versions from 1.7.7.0 ~ 1.7.7.8 to (1.7.8.11 - 8.2.0) on PHP 7.1 ~ 7.3
| Type?             | improvement
| Sponsor company   | @PrestaShopCorp
| How to test?      | CI is :green_apple:

### CODE REVIEW

**Comment by nesrineabdmouleh** [comment]
issue => https://github.com/PrestaShop/PrestaShop/issues/36425

**Review by Progi1984 — DISMISSED** [review]

**Review by Progi1984 — DISMISSED** [review]

**Review by Progi1984 — APPROVED** [review]

**Comment by ps-jarvis** [comment]
PR merged, well done!

Message to @PrestaShop/committers: do not forget to milestone it before the merge.

**Pull request merged**
